coops-ui/src/components/Common/SummaryDirectors.vue

```diff
@@ -208,9 +208,9 @@ export default class SummaryDirectors extends Mixins(DateMixin, EntityFilterMixi
   EntityTypes: {} = EntityTypes
 
   /**
-    * On-Created hook to create two new arrays of directors for summary output
-    * One array will contain new/changed directors
-    * One array will contain all ceased directors
+    * On-Created hook to create two new arrays of directors for summary output.
+    * - "directorSummary" will contain new/changed directors
+    * - "directorsCeased" will contain all ceased directors
```

Comment:
Was the method itself changed in another PR?

Reply:
No, I simply updated the comment. PR #577 actually changes this method. I'm expecting a conflict but this way I will remember to update the comment :)

Reply:
Changed reverted.